models/marts/finance/fct_orders.sql

```diff
@@ -1,6 +1,7 @@
 {{
     config(
-        tags = ['finance']
+        tags = ['finance'],
+        materialized='view'
```

Comment:
Do you do this just to show it overrides the config on the dbt_project.yml file?

Reply:
Yeah, I will change that back though! This slipped through.